Makefile

```diff
@@ -0,0 +1,20 @@
+## Location to install dependencies to
+LOCALBIN ?= $(shell pwd)/bin
+$(LOCALBIN):
+	mkdir -p $(LOCALBIN)
+
```

Comment:
Should we copy the target that overwrites values in `support/defaults.go`?

Reply:
Thank you @astefanutti , yes I can move that over. Is it okay to remove that make target from CFO entirely? From a look at the use of the values in defaults.go having them just in the new repo should suffice.

Reply:
Yes I think so too. There are also environment variables that can be used to overwrite these defaults.

Reply:
Ok I will remove it then. Thank you. 
I also need some input on reviewers/approvers for the repo - will I copy these over from CFO as they are there?
And should I add any PR checks/actions at this point?

Reply:
Yes we can do that in a separate PR if you want. We'll also have to setup openshift-ci for the repo.

Reply:
Ok I will create a follow up issue for these. Thank you!

Reply:
Follow on issue: https://github.com/project-codeflare/codeflare-common/issues/2 
I've added the `needs-triage` label but have not added it to any sprint.